pages/reset/index.jsx

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from 'react'
+import { classNames } from '@oacore/design/lib/utils'
+
+import styles from './styles.module.css'
+
+import { Card, TextField, Button } from 'design'
+import { withGlobalStore } from 'store'
+
+const ResetStep1 = ({ className, store, ...restProps }) => {
+  const [message, setMessage] = useState(null)
+
+  // TODO: Better texts and move it to @oacore/texts
+  const handleSubmit = async (event) => {
+    event.preventDefault()
+    try {
+      const formData = new FormData(event.target)
+      const data = {
+        email: formData.get('email'),
+      }
+      await store.user.requestResetToken(data)
+      setMessage('Please check your email for following instructions.')
+    } catch (error) {
+      setMessage("Something wen't wrong. Please try it again!")
```

Comment:
I am not sure it's good to use exclamation mark here. @mcancellieri do you have any ideas?

Reply:
I didn't like it too but using dot is not correct too I guess. As I said these phrases should be carefully reviewed and changed at some point.